api/src/pcapi/core/offers/repository.py

```diff
@@ -658,10 +658,7 @@ def get_offer_by_id(offer_id: int) -> models.Offer:
             .options(joinedload(models.Offer.product, innerjoin=True))
             .options(joinedload(models.Offer.priceCategories).joinedload(models.PriceCategory.priceCategoryLabel))
             .options(
-                joinedload(
-                    models.Offer.venue,
-                    innerjoin=True,
-                ).joinedload(
+                joinedload(models.Offer.venue, innerjoin=True,).joinedload(
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick:** ‏
```suggestion
                joinedload(models.Offer.venue, innerjoin=True).joinedload(
```

Reply:
encore un coup de ce plaisantin de `black` ça...
je corrige ;)